python/tests/test-export-tagger.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+BASELINE_DIR=${HOME}/dev/work/baseline
+TAGGER_MODEL=tagger-24429.zip
+TEST_FILE=eng.testb.small
+TEST_LOAD=$TEST_FILE.load
+TEST_SERVE=$TEST_FILE.serve
+TEST_SERVE_PREPROC=$TEST_FILE.serve_preproc
+NUM_FEATURES=3
+SLEEP=5
+
+docker stop tfserving
+docker rm tfserving
+[ -e $TAGGER_MODEL ] && rm $TAGGER_MODEL
+[ -e $TEST_FILE ] && rm $TEST_FILE
+[ -e $TEST_LOAD ] && rm $TEST_LOAD
+[ -e $TEST_SERVE ] && rm $TEST_SERVE
+[ -e $TEST_SERVE_PREPROC ] && rm $TEST_SERVE_PREPROC
+
+echo "running test for tagger"
+echo "------------------------"
+
+echo "downloading trained tagger model and test file"
+wget https://www.dropbox.com/s/86xzsofbgk3dzyj/tagger-24429.zip?dl=1 -O $TAGGER_MODEL
+wget https://www.dropbox.com/s/pp3pcq5q5ko1df6/eng.testb.small?dl=1 -O $TEST_FILE
+
+echo "tagging by loading the model"
+python $BASELINE_DIR/api-examples/tag-text.py --model $TAGGER_MODEL --text $TEST_FILE  > $TEST_LOAD
+sleep $SLEEP
+
+echo "exporting w/o preproc"
+MDIR=models/conll
+rm -rf $MDIR
+mkdir -p $MDIR
+mead-export --config $BASELINE_DIR/python/mead/config/conll.json --model $TAGGER_MODEL --is_remote false --output_dir $MDIR
+sleep $SLEEP
+
+echo "running tf serving"
+docker stop tfserving
```

Comment:
`> /dev/null 2>&1` will suppress the error of not finding the container which will help clean up this output.